app/src/main/res/layout-land/submitted_answer_item.xml

```diff
@@ -84,9 +88,15 @@
       android:src="@drawable/ic_check_primary"
       android:visibility="@{viewModel.isCorrectAnswer &amp;&amp; !viewModel.hasConversationView ? View.VISIBLE : View.GONE}"
       app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintHorizontal_bias="0.0"
-      app:layout_constraintStart_toEndOf="@+id/submitted_answer_container"
+      app:layout_constraintStart_toEndOf="@+id/submitted_answer_text_view"
       app:layout_constraintTop_toTopOf="parent" />
-  </LinearLayout>
+
+    <androidx.constraintlayout.widget.Guideline
+      android:id="@+id/tick_guideline"
```

Comment:
This should probably be called ``answer_tick_guideline`` to be a bit closer to ``answer_tick`` above. Ditto for other layout files.

Reply:
Done